src/test/java/uk/nhs/ctp/util/SeleniumUtil.java

```diff
@@ -3,22 +3,60 @@
 import java.io.File;
 import java.io.FileOutputStream;
 import lombok.experimental.UtilityClass;
+import org.checkerframework.checker.nullness.compatqual.NullableDecl;
+import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.OutputType;
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 @UtilityClass
 public class SeleniumUtil {
 
   public void takeScreenshot(WebDriver driver, String filename) {
     new File(("target/screenshots/")).mkdirs();
     try {
-      FileOutputStream out = new FileOutputStream("target/screenshots/screenshot-" + filename + ".png");
-      out.write( ((TakesScreenshot)driver).getScreenshotAs(OutputType.BYTES));
+      FileOutputStream out = new FileOutputStream(
+          "target/screenshots/screenshot-" + filename + ".png");
+      out.write(((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES));
       out.close();
     } catch (Exception ignored) {
 
     }
   }
 
+  /**
+   * Sends value to an input element using sendKeys and waits to make sure the value has been accepted
+   */
+  public void setValue(WebElement element, String value, WebDriverWait wait) {
+    element.sendKeys(value);
+    wait.until(ExpectedConditions.textToBePresentInElementValue(element, value));
+  }
+
+  /**
+   * Condition to check if an element is not displayed, or cannot be found in the DOM
+   */
+  public ExpectedCondition<Boolean> notVisible(WebElement element) {
```

Comment:
```suggestion
  public ExpectedCondition<Boolean> absenceOf(WebElement element) {
```
Just so that it's not confused with `ExpectedConditions.invisibilityOf() ` as they are doing subtly different things?

Reply:
Good point, I tried invisibilityOf first but due to the selector not finding any element it throws an exception.